sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/PiiEntityDomainType.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+/**
+ * Defines values for PiiEntityDomainType.
+ */
+public enum PiiEntityDomainType {
+    /**
+     * Protected health information (PHI) as the PiiEntityDomainType.
+     */
+    PROTECTED_HEALTH_INFORMATION("PHI");
```

Comment:
Does this enum just have one value?
If this enum is forseen to hold more values in the future we should consider updating it to an expandable enum?

Reply:
yes. updated to expandable enum because of the enum value set can grow